configs/det/dbnet/db++_r50_icdar15.yaml

```diff
@@ -134,8 +134,13 @@ eval:
           img_mode: RGB
           to_float32: False
       - DetLabelEncode:
-      - ScalePadImage:
-          target_size: [ 1152, 2048 ] # h, w
+      #- ScalePadImage:
+      #    target_size: [ 1152, 2048 ] # h, w
+      - DetResize: 
+          target_size: [ 1152, 2048]
+          keep_ratio: True
+          limit_type: auto
```

Comment:
I feel this is a bit confusing. When I want an image to have a specific resolution (set by `target_size`) and keep its ratio, I must need to set `limit_type` to `auto`. Otherwise, I will get completely unexpected output.

Reply:
Don't have to set limit_type auto in this case. You can just use the default "min". Here auto is just to make the same as ScalePadImage.

Reply:
If I don't set `limit_type` to `auto`, the output will be of size 736 by the shortest side.